packages/sdk/src/oracle/common/store/store.ts

```diff
@@ -0,0 +1,49 @@
+import { RawStore } from "./rawStore";
```

Comment:
this file wasnt modified other than to implement the new interfaces

Reply:
Maybe this is obvious, but why do we have to copy these files? I would've though they could've been made more generic or functionality could have been added.